chain-impl-mockchain/src/setting.rs

```diff
@@ -310,6 +307,10 @@ impl Settings {
         params
     }
 
+    pub fn to_treasury_params(&self) -> TaxType {
```

Comment:
remove `to_`, I think the convention around `to_` methods is that they take `self`